assets/dev/scss/frontend/widgets/link-in-bio.scss

```diff
@@ -0,0 +1,93 @@
+.elementor-widget-link-in-bio {
+	--align-self: center;
+	--e-link-in-bio-gutter: 2em;
+
+	.elementor-widget-container {
+		border: 1px solid blue;
```

Comment:
- Is it really `blue`? Is there maybe a variable you can use from `elementor/assets/dev/scss/helpers/variables.scss`
- Can you maybe style the wrapper div and not `elementor-widget-container`? We might want to remove this div at some point.

Reply:
@willhowat can you check this?

Reply:
@louiswol94 @nicoladj77 this as with other CSS in this PR is pure placeholder and will be aligned in a later PR.
Noted on switching wrappers, thanks for the heads up

Reply:
@louiswol94 this has now been addressed in this PR with that latest updates